packages/select/src/utils/getStyles.ts

```diff
@@ -220,13 +227,17 @@ export const getStatesStyles: GetStyles = memoizee((
   const focusStyles = getAppearanceStyleProperty(theme, appearance, baseAppearance, 'focus');
   const selectedStyles = getAppearanceStyleProperty(theme, appearance, baseAppearance, 'selected');
   const disabledStyles = getAppearanceStyleProperty(theme, appearance, baseAppearance, 'disabled');
+  const filledStyles = getAppearanceStyleProperty(theme, appearance, baseAppearance, 'filled');
 
   return memoizee((
     elementName,
-    isDisabled,
-    isFocused,
-    isSelected,
-    isSearchable,
+    {
```

Comment:
This construction violates memoization, because the second argument will always be a new object, and the memoization will not hit the cache.